Accessories/uInverter/DESIGN CALCULATIONS.md

```diff
@@ -0,0 +1,45 @@
+﻿## uInverter Design Calculations
+
+### Core Components
+ #### DC Barrel Jack
+ 
+ The 2.1mm [DC barrel jack](https://www.digikey.com/en/products/detail/3643/1528-2553-ND/8346574) is rated for 15Vdc and is supplied from a standard [AC-DC wall adapter](digikey.com/en/products/detail/cui-inc/SWM30-12-NV-P5/7388285) rated for 12V. 
+
+#### DC link 
+A 3mF capacitor is selected as a DC link capacitor to protect the voltage ripple from input to the components comprising the half bridge. The capacitor is should be able to withstand a 9V input voltage. 
+
+Using formula 
+![](http://ec.kemet.com/wp-content/uploads/sites/4/2019/10/dc-link-design-tips-04-300x70.png)
+ * where _Vp(max)_ is chosen to be 0.5V
+ * duty cycle _(dc)_ assumed to be 0.75
+ * Switching frequency _fsw_ is 1kHz
+ * _Iout_ is a maximum of 7.5A 
+
+we can use any capacitor >2.8mF. 
+
+#### Half Bridges
+
+The half bridge inverter is designed using [UCC27531 MOSFET gate drivers](https://www.ti.com/lit/ds/symlink/ucc27531.pdf?HQS=TI-null-null-mousermode-df-pf-null-wwe&ts=1603960573394) with an On-State maximum current of 2.5 A. This limits the current in each phase of the inverter to 2.5A and sets the maximum input current drawn to 7.5A.
+
+The gate driver uses the 12V input voltage as VDD, which sits in the defined range of 10V-35V. As per the Figure on Page 41 of the datasheet, OUTH and OUTL are connected by a nominal 1ohm resistor. 
+
+#### R-L Load and Current Sense Resistor
+
+Each phase of the inverter is connected to an R-L load and a sense resistor. Since each phase receives 50% of the bus voltage, i.e., 6V and 2.5A as regulated by the driver used in the half-bridge. 
+
+Given that the frequency of signal is 1kHz, a 300uH resistor and a 400mOhm resistor is chosen, where the resistor doubles up as the Rsense. The 400 mOhm resistor will be justified in the next section, and the impedance of the inductor (Z= 2* *pi* **f* *L) is 0.63 ohm.
```

Comment:
> impedance of the inductor (Z= 2* *pi* **f* *L)  is 0.63 ohm

Check your math on this....

Also, please more fully explain your choice of L.